MeepMeeptest/src/main/java/com/example/meepmeeptest/MeepMeeptest.java

```diff
@@ -0,0 +1,32 @@
+package com.example.meepmeeptest;
```

Comment:
It looks like extra stuff unrelated to the ticket was checked in, please remove/undo